website/cue/reference/administration/downloads.cue

```diff
@@ -127,6 +127,11 @@ administration: {
 						arch:   "x86_64"
 						extra:  "apple-darwin"
 					},
+				  {
+						target: "arm64-apple-darwin-tar-gz"
+						arch:   "ARM64"
+						extra:  "apple-darwin"
+					}
```

Comment:
This will 404 on all existing releases. I'm wondering if we need to add some sort of optional version specifier here. Like: `version: ">= 0.44.0"` that could be used when rendering the download page to determine if a particular platform should be rendered. This could be follow-up work to this PR (in which case we should just delete this new entry for now to avoid the user confusion from 404s).